### ISSUE TITLE
🐛Delay after battlefield win and player action issue

### ISSUE THREAD

**Issue by TracentEden** [DerpyProjectGroup/topaz#3190]
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my issue will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated.

## Branch affected by issue 

`base`

## Steps to reproduce
Enter a mission battlefield (coded under old battlefield system) such as Ancient Flames Beckon, beat the boss, and observe a 15 second delay before the winning CS starts and observe that the player can move and perform actions during this delay

## Expected behavior
The winning CS should start quicker after defeating the boss (as in retail, see [here](https://youtu.be/IKf8CGSMHww?t=304)) and the player should not be able to move or perform actions during this short delay otherwise you can have strange mischief such as exiting the battlefield after winning but before the winning CS (which is currently possible).

The delay issue is due to the cutsceneTimer variable in the old battlefield system in battlefield.lua which applies the 15 sec wait to all battlefields. In the new battlefield system this time defaults to 5 seconds (rather than 15) and can be set per battlefield (though all seem to be using the default at the moment). So maybe a fix could be waiting until all battlefields are ported to new system and going through to set the time on a per battlefield or battlefield type basis. While preventing players from performing actions and moving during the delay might need a separate fix in battlefield system.

(also impacts ASB)

**Comment by jmcmorris** [comment]
Are you unable to perform actions on retail after winning a battlefield?

I do think a valid fix for this is to just wait for the battlefields to be ported. They are actively being ported so it will hopefully not be too long.

**Comment by TeoTwawki** [comment]
> Are you unable to perform actions on retail after winning a battlefield?

Yes, because everything stops instantly on retail. way back in dsp this delay was a side effect of fixing another more serious bug. it just never got revisited. Our mobs don't instant stop like on retail either and anything still alive will get in another swing or 2.
